assests/javascript/index.js

```diff
@@ -1 +1,225 @@
+          <i class="fa fa-solid fa-wind main-icon"></i>
+          <figcaption class="main-icon-detail">${convertMeterPerSecondToMilesPerHour(
+            curentWeather.wind.speed
+          )}</figcaption>
+        </figure>
+      </li>
+      <li>
+        <figure class="main-icon-container">
+          <i class="fa fa-solid fa-droplet main-icon"></i>
+          <figcaption class="main-icon-detail">${
+            curentWeather.main.humidity
+          }%</figcaption>
+        </figure>
+      </li>
+    </ul>
+  </section>
+  <ul class="forecasting-weather-list">
+    <li>
+      <figure class="forecasting-weather-box">
+        <img
+          alt="forecasting weather image"
+          src="http://openweathermap.org/img/wn/${
+            dailyForecastingWeatherList[0].weather[0].icon
+          }@2x.png"
+          class="forecasting-weather-icon"
+        />
+        <figcaption class="forecasting-weather-text-box">
+          <p class="forecasting-weather-paragraph">${convertKelvinToCelsius(
+            dailyForecastingWeatherList[0].main.temp
+          )}</p>
+          <p>Today</p>
+        </figcaption>
+      </figure>
+    </li>
+    <li>
+      <figure class="forecasting-weather-box">
+        <img
+          alt="forecasting weather image"
+          src="http://openweathermap.org/img/wn/${
+            dailyForecastingWeatherList[1].weather[0].icon
+          }@2x.png"
+          class="forecasting-weather-icon"
+        />
+        <figcaption class="forecasting-weather-text-box">
+          <p class="forecasting-weather-paragraph">${convertKelvinToCelsius(
+            dailyForecastingWeatherList[1].main.temp
+          )}</p>
+          <p>${
+            convertTimeStampToDate(dailyForecastingWeatherList[1].dt).dayOfWeek
+          }</p>
+        </figcaption>
+      </figure>
+    </li>
+    <li>
+      <figure class="forecasting-weather-box">
+        <img
+          alt="forecasting weather image"
+          src="http://openweathermap.org/img/wn/${
+            dailyForecastingWeatherList[2].weather[0].icon
+          }@2x.png"
+          class="forecasting-weather-icon"
+        />
+        <figcaption class="forecasting-weather-text-box">
+          <p class="forecasting-weather-paragraph">${convertKelvinToCelsius(
+            dailyForecastingWeatherList[2].main.temp
+          )}</p>
+          <p>${
+            convertTimeStampToDate(dailyForecastingWeatherList[2].dt).dayOfWeek
+          }</p>
+        </figcaption>
+      </figure>
+    </li>
+  </ul>`;
+}
+
+function showErrorMassage() {
+  mainSection.innerHTML += `<p class="error-message">Somthing went wrong. Please refresh the page!</p>`;
+}
+
+function convertTimeStampToDate(date) {
+  const dateTime = new Date(date * 1000);
+
+  return {
+    year: dateTime.getFullYear(),
+    month: convertMonthNumberToName(dateTime.getMonth()),
+    dayOfMonth: dateTime.getDate(),
+    dayOfWeek: convertDayOfWeekNumberToName(dateTime.getDay()),
+  };
+}
```

Comment:
Read about this and remove unnecessary logic here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Reply:
that was awesome.thankssss